Katello/katello
# Fixes #33399 - Add errata by date filter rule

To test:

On new CV UI > Create a new filter (Errata - Date and Type)
The filter rule page should appear:

![Screenshot from 2021-09-30 19-04-33](https://user-images.githubusercontent.com/21146741/135541074-7ff49032-ec4a-4680-955a-3f8b3419036d.png)
